google-cloud-spanner/src/test/java/com/google/cloud/spanner/spi/v1/GfeLatencyTest.java

```diff
@@ -238,38 +234,31 @@ public void testGfeLatencyExecuteSql() throws InterruptedException {
     long latency =
         getMetric(
             SpannerRpcViews.SPANNER_GFE_LATENCY_VIEW,
-            projectId,
-            instanceId,
-            databaseId,
             "google.spanner.v1.Spanner/ExecuteSql",
             false);
-    assertEquals(fakeServerTiming.get(), latency);
+    assertEquals(FAKE_SERVER_TIMING.get(), latency);
   }
 
   @Test
   public void testGfeMissingHeaderCountExecuteStreamingSql() throws InterruptedException {
     try (ResultSet rs = databaseClient.singleUse().executeQuery(SELECT1AND2)) {
       rs.next();
     }
+    Stopwatch watch = Stopwatch.createStarted();
     long count =
         getMetric(
             SpannerRpcViews.SPANNER_GFE_HEADER_MISSING_COUNT_VIEW,
-            projectId,
-            instanceId,
-            databaseId,
             "google.spanner.v1.Spanner/ExecuteStreamingSql",
             false);
     assertEquals(0, count);
+    System.out.println("Duration: " + watch.elapsed());
```

Comment:
is this intentional?

Reply:
good catch, nope, leftover from figuring out where it was slow.